frontend/src/util/shared/survey.ts

```diff
@@ -11,28 +11,48 @@ export function mapToList<T>(map: Record<string, T>): SurveyOption[] {
 // count: number of students who chose the option
 export type OptionCount = {
     option: string,
-    count: number
+    count: number,
+    students: string[],
+    optionExists: boolean,
 }
 
 export default function formatSurveyResponses(options: SurveyOption[], responses: Record<string, string[]>): OptionCount[] {
+    // Map from option to number of students who chose that option
```

Comment:
we should combine the two maps into one. the count map can be derived from the student map. two makes it easy to miss updating one of them